pymysqlreplication/row_event.py

```diff
@@ -710,6 +761,20 @@ def _dump(self):
         if self.__optional_meta_data:
             self.optional_metadata.dump()
 
+    def _to_json(self):
+        result_dict = {
+            "event_name": type(self).__name__,
+            "table_id": self.table_id,
+            "schema": self.schema,
+            "table": self.table,
+            "columns": self.column_count,
+        }
+
+        if self.optional_metadata:
+            result_dict.update(self.optional_metadata.to_json())
```

Comment:
self.optional_metadata always got object 
optional_metadata = OptionalMetaData()

try this variable  self.__optional_meta_data

and can we get some testcases?